github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Upcloud go api test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ 1.14.x, 1.15.x ]
+    steps:
+      - name: Test
```

Comment:
Oh yeah. This steps name should probably be `setup` or `install` or `init env`. Since that is what the step is doing

Reply:
Did some changes (some cleaning up), any thoughts?

Reply:
could/should probably rename the build to test (also the script), but left like this for now

Reply:
naming steps is IMO a good practice. At least when the step is doing complicated or unusual stuff. This seems ok to me though :)